packages/jitm/src/Manager.php

```diff
@@ -337,29 +384,32 @@ function get_messages( $message_path, $query ) {
 				continue;
 			}
 
-			JetpackTracking::record_user_event( 'jitm_view_client', array(
-				'jitm_id' => $envelope->id,
-			) );
+			\JetpackTracking::record_user_event(
+				'jitm_view_client',
+				array(
+					'jitm_id' => $envelope->id,
+				)
+			);
 
-			$normalized_site_url = Jetpack::build_raw_urls( get_home_url() );
+			$normalized_site_url = \Jetpack::build_raw_urls( get_home_url() );
```

Comment:
What are we planning to do with references to the `Jetpack` class, considering that packages are supposed to be independent?

Reply:
We'll continue to iterate on it and remove dependencies in another PR once it's been merged into the feature branch.